pkgs/os-specific/linux/kernel-headers/default.nix

```diff
@@ -24,7 +24,7 @@ let
       flex bison python rsync
     ];
 
-    extraIncludeDirs = lib.optional stdenvNoCC.hostPlatform.isPowerPC ["ppc"];
+    extraIncludeDirs = lib.optional stdenvNoCC.hostPlatform.isPower32BigEndian ["ppc"];
```

Comment:
This is a bit bizarre and I have no clue what this is supposed to be doing, so let's not touch it. Interestingly the only dir this could be referring to is `sound/ppc` which is a bit confusing to me.

@Ericson2314 `extraIncludeDirs` is only used here in `nixpkgs`, maybe we can remove it?